test/operator-framework-e2e/operator_framework_test.go

```diff
@@ -6,125 +6,86 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-
 	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/rand"
-	"k8s.io/client-go/rest"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	catalogd "github.com/operator-framework/catalogd/api/core/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	operatorv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
 )
 
-var (
-	cfg *rest.Config
-	c   client.Client
-	ctx context.Context
-)
-
 func TestOperatorFramework(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Operator Framework E2E Suite")
-}
+	t.Parallel()
+	cfg := ctrl.GetConfigOrDie()
```

Comment:
Isn't the `ctrl.GetConfigOrDie()` helper for in-cluster config when running in a Pod? Why is it in use here?

Reply:
I believe it will also build the configuration from the KUBECONFIG when run outside of a cluster

Reply:
As long as you can run it locally that's fine